### ISSUE TITLE
RFE Chromecast support

### ISSUE THREAD

**Issue by kim0** [yify-pop/yify-pop#38]
I apologize for not yet testing this (my chromecast device is still on its way). Wanted to know if CC is supported out of the box.

**Comment by inkubux** [comment]
I would love chromecast support. Castnow is able to cast from peerflix with subtitles, you might want to look there.

**Comment by jkaberg** [comment]
Isn't yify-pop using video-js?

https://github.com/benjipott/video.js-chromecast

**Comment by philipgiuliani** [comment]
Yes it is using video-js. Im the author of the plugin, if you need anything special just contact me :)

**Comment by Oliver84** [comment]
So is Chromecast working for yify-pop? Do we just need to add data-cast-api-enabled="true" to the html and videojs.chromecast.css and videojs.chromecast.js to the head?

**Comment by philipgiuliani** [comment]
Yes and enable the plugin in the videojs config. Everything is described there. VideoJS 5 is not supported yet.
